packages/core/src/components/date-picker/utils/month-range.ts

```diff
@@ -0,0 +1,38 @@
+import defaultLocalization from "../date-localization";
+
+export function monthRange(selectedYear: number, minDate?: Date, maxDate?: Date): string[] {
+  if (minDate && maxDate) {
+    const { minYear, minMonth } = { minYear: minDate.getFullYear(), minMonth: minDate.getMonth() };
+    const { maxYear, maxMonth } = { maxYear: maxDate.getFullYear(), maxMonth: maxDate.getMonth() };
+
+    return defaultLocalization.monthNames.filter((_month, index) => {
```

Comment:
Ik stel voor dat we `this.localization` meegeven aan deze functie, zodat die private property in `datepicker.tsx` zijn 'waarheid' behoudt.

Reply:
Klinkt goed